openmdao/docs/openmdao_book/other_useful_docs/developer_docs/signing_commits.ipynb

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
<p>I had to <code>git config user.signingkey XXXXXXXX</code></p><p> </p><p>https://stackoverflow.com/questions/36810467/git-commit-signing-failed-secret-key-not-available</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>fair enough.. I was just following your outline (and clicking through to the links) but missed that step</p>